doc/usecases/map_timeline.md

```diff
@@ -0,0 +1,26 @@
+# Use Case: Map Timeline View
+
+## Summary
+
+- **Scope:** Map View
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can view the map at different points in time by using the timeline feature
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  The user has opened the app and selected the map view.
+- **Main success scenario:**
+  The user selects the timeline view and uses the scroll bar with a month granularity or date field to navigate to a different point in time. 
+  The map updates to show the state of the garden at the selected time.
+- **Alternative scenario:**
+  The user tries to navigate to a date that is outside the range of the timeline. 
+  In this case, the app displays an error message indicating that the requested date is not available.
```

Comment:
What is the range? For reasonable dates it must show empty map, the user might want to draw there.

Reply:
Hm, I'm not sure here. Do you have a suggestion on how to handle this?

Reply:
Difficult to judge without trying it out. Yvonne was thinking about only showing 1,5 years in total in the timeline, I was thinking about all events that have ever happened (can be e.g. 10 years) and 1 year additionally at beginning and end.

What definitely would be nice is to get mouseover hints which month would be selected on a click.